rs/ledger_suite/icrc1/index-ng/tests/incompatible_token_type.rs

```diff
@@ -0,0 +1,88 @@
+use crate::common::{
+    account, default_archive_options, install_ledger, wait_until_sync_is_completed,
+};
+use candid::{Encode, Principal};
+use ic_agent::identity::Identity;
+use ic_icrc1_index_ng::{IndexArg, InitArg as IndexInitArg};
+use ic_icrc1_test_utils::minter_identity;
+use ic_state_machine_tests::StateMachine;
+
+mod common;
+
+fn index_wasm_u64() -> Vec<u8> {
+    let index_wasm_path = std::env::var("IC_ICRC1_INDEX_WASM_U64_PATH").expect(
+        "The Ledger wasm path must be set using the env variable IC_ICRC1_INDEX_WASM_U64_PATH",
+    );
+    std::fs::read(&index_wasm_path).unwrap_or_else(|e| {
+        panic!(
+            "failed to load Wasm file from path {} (env var IC_ICRC1_INDEX_WASM_U64_PATH): {}",
+            index_wasm_path, e
+        )
+    })
+}
+
+fn index_wasm_u256() -> Vec<u8> {
+    let index_wasm_path = std::env::var("IC_ICRC1_INDEX_WASM_U256_PATH").expect(
+        "The Ledger wasm path must be set using the env variable IC_ICRC1_INDEX_WASM_U256_PATH",
+    );
+    std::fs::read(&index_wasm_path).unwrap_or_else(|e| {
+        panic!(
+            "failed to load Wasm file from path {} (env var IC_ICRC1_INDEX_WASM_U256_PATH): {}",
+            index_wasm_path, e
+        )
+    })
```

Comment:
Duplicated logic. Can we factor into a common function that has the env var as parameter?

Reply:
Done!